src/timearray.jl

```diff
@@ -259,60 +261,89 @@ function print_time_array(io::IO, ta::TimeArray{T}, short=false) where T
     spacetime = textwidth(string(ts[1]))
     pages = _showpages(dcol, spacetime, colwidth)
 
-    for p ∈ pages
-        # row label line
-        ## e.g. | Open  | High  | Low   | Close  |
-        print(io, "│", " "^(spacetime + 2))
-        for (name, w) in zip(colnames(ta)[p], colwidth[p])
-            print(io, "│ ", _rpad(name, w + 1))
-        end
-        println(io, "│")
-        ## e.g. ├───────┼───────┼───────┼────────┤
-        print(io, "├", "─"^(spacetime + 2))
-        for w in colwidth[p]
-            print(io, "┼", "─"^(w + 2))
-        end
-        print(io, "┤")
-
-        # timestamp and values line
-        if nrow > (drow - res_row)
-            for i in tophalf
-                println(io)
-                print(io, "│ ", ts[i], " ")
-                for j in p
-                    print(io, "│ ", _rpad(strs[i, j], colwidth[j] + 1))
-                end
-                print(io, "│")
-            end
+    _print_page(io, pages[1], ta, 
+        spacetime, colwidth, nrow, drow, res_row, tophalf, bothalf, 
+        length(pages))
+    
+    if length(pages) > 1
```

Comment:
I think we need to support `@show` for printing all pages out.

Reply:
Yes, that would be great and it would take advantage of the paging machinery. I'm not sure how to do this, though; I tried looking at the `DataFrames` package and to `@show df` with a large `DataFrame` and it dumps all the data on screen (not readable) and then the same trimmed view of the `df`. I am not sure of what methods to override. Or maybe a separate method for this?

Reply:
Oh, I guess `DataFrames` use the `:limit` option of  `IOContext` to control the omitting.

You can try this:
```
show(IOContext(stdout, :limit => false), df)
```

vs 

```
show(df)
```

I think we can also utilize that option.